src/adapter/activateDaffodilDebug.ts

```diff
@@ -209,19 +259,43 @@ export function activateDaffodilDebug(
         createDebugRunFileConfigs(resource, 'run', 'execute')
       }
     ),
-    vscode.commands.registerCommand('extension.dfdl-debug.copyTDML', (_) => {
-      // Ask for destination path
-      // Copy file in /tmp to destination path
-      // TDMLConfig.path should not be used here because that only matters when sending to the server
-      // We could make it so that if someone wants to specify the path and set the action to 'copy', but
-      //   that doesn't make a whole lot of sense.
-      let targetResource = vscode.commands.executeCommand(
-        'extension.dfdl-debug.getTDMLPath'
-      )
+    vscode.commands.registerCommand(
+      'extension.dfdl-debug.copyTDML',
```

Comment:
Question: Why is this command available to the user? It assumes some TDML file exists at some path... I just don't get it.

Reply:
This functionality is implemented based on the design discussion listed [here](https://github.com/ctc-oss/daffodil-vscode/discussions/126) and described below,

- We should not need to specify a location for the TDML file. We should always put it in /tmp (or OS equivalent) under a static name
- We should provide 'Cleanup TDML File' and 'Copy TDML File' actions on the command palette.
- Copy TDML File should copy the existing file to the user-defined location under $workspace.
- There should be an Append/Copy option that takes the test case from the file under /tmp and appends it to the user-defined location under $workspace.

Reply:
Thanks for the link and summary, it makes sense now.